frontend/src/app/testResults/__snapshots__/TestResultsList.test.tsx.snap

```diff
@@ -69,6 +69,7 @@ exports[`TestResultsList should render a list of tests 1`] = `
               <tbody>
                 <tr
                   class="sr-test-result-row"
+                  data-patient-link="https://simplereport.gov//pxp?plid=68c543e8-7c65-4047-955c-e3f65bb8b58a"
```

Comment:
It'll work but might want to look into removing the redundant forward slash

Reply:
:thinking: So, I'm thinking this is just a test setup issue - if you look at the `getUrl` function, it just reads the env vars `REACT_APP_BASE_URL` and `PUBLIC_URL`, so I'm guessing one of those contains a slash in the test env when it shouldn't. But since these are just snapshots and not actually going to be clicked, I don't think it matters, right?

Reply:
We ended up fixing this by changing `getUrl` to only append a slash if it's missing from the env vars